conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/TezosTypes.scala

```diff
@@ -14,7 +20,7 @@ object TezosTypes {
 
   //TODO use in a custom decoder for json strings that needs to have a proper encoding
   lazy val isBase58Check: String => Boolean = (s: String) => {
-    val pattern = "^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]*$".r.pattern
+    val pattern = "^[^\\W0OlI_]*$".r.pattern
```

Comment:
are you sure this pattern negates every character that shouldn't be there? Shouldn't you use [a-zA-Z1-9], so you wouldn't have any special characters and then remove specific letters/numbers?

Reply:
Yea, it should work as desired; all tests are passing as well. We negate all non-word symbols as well as underscore (which is captured by `\w`), plus the symbols that can be mismatched, namely - `I`, `l`, `O`, and `0`. In case it's ambiguous, maybe I should simply leave it as it was originally?

Reply:
I would leave it as it was, as it was not the purpose of this ticket - in the future refactor we can think of changing it and additional tests.

Reply:
sure, removed it